projects/goharbor/harbor/docker/linux/harbor-redis/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+RUN amazon-linux-extras enable redis6 \
+    && yum install -y redis >> /dev/null \
+    && yum clean all
+
+VOLUME /var/lib/redis
+WORKDIR /var/lib/redis
+
+COPY _output/make/photon/redis/docker-healthcheck /usr/bin/
+COPY --chown=redis:redis _output/make/photon/redis/redis.conf /etc/redis.conf
+# COPY _output/LICENSES /LICENSES
+# COPY ATTRIBUTION.txt /ATTRIBUTION.txt
+
+RUN chmod +x /usr/bin/docker-healthcheck
+
+HEALTHCHECK CMD ["docker-healthcheck"]
```

Comment:
For this one do we really need the healthcheck script?  Kubernetes isnt going to call this one anyway right?  If we dont need this, can we use the minimal base here and install redis like we did [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/redis/redis/docker/linux/Dockerfile).